dianna/methods/kernelshap_image.py

```diff
@@ -100,7 +104,7 @@ def explain(
         https://scikit-image.org/docs/dev/api/skimage.segmentation.html#skimage.segmentation.slic
 
         Returns:
-            Explanation heatmap of shapley values for each class (np.ndarray).
+            Explanation heatmap for each class (np.ndarray).
```

Comment:
I think I would prefer the old comment:
```suggestion
            Explanation heatmap of Shapley values for each class (np.ndarray).
```